### DISCUSSION TITLE
Add a prop to keep whitespace in <Text /> component

### DISCUSSION THREAD

**Discussion by jannikac** [mantinedev/mantine#7127] [Feature requests]
Hi, I find myself often running into the problem that when I want to display text that a user entered, line breaks and whitespace are dropped. This is default behaviour with `p` or `span` `tags` and you'd need to use either a `pre` tag or add the style `white-space: "pre-wrap"`. This is what im currently doing. 

I think it would be useful to add a property to the `Text` component of mantine like `keepWhitespace` or `preserveWhitespace` that either changes to root component to a `pre` tag or uses the `white-space: "pre-wrap"` style.

I can help with implementing if you want me to :)

**Comment by rtivital** [comment]
You can use `<Box component="pre" />` for that. Text component wasn't really meant to be used this way.

**Reply by jannikac** [reply]
Okay, thanks for the reply.
